model.py

```diff
@@ -894,8 +894,7 @@ def _calculate_until(
         # Start with the default loan period to clear out everyone who
         # currently has the book checked out.
         duration = default_loan_period
-
-        if queue_position < total_licenses:
+        if queue_position <= total_licenses:
```

Comment:
I think the only behavior that actually changed here is when `queue_position` and `total_licenses` are both 1. If `total_licenses` is > 1 and the same as `queue_position`, `cycles` was already 0 in the else condition.

That does seem like it was wrong before, but I think it might still be wrong - it was `default_loan_period + default_loan_period + default_reservation_period`, and now it's `default_loan_period`, but I think it should be `default_loan_period + default_reservation_period`. In the worst case, you're at position 1 in the queue and another patron has the book reserved but hasn't checked it out yet.